src/test/scala/com/mozilla/telemetry/streaming/TestErrorAggregator.scala

```diff
@@ -317,8 +322,55 @@ class TestErrorAggregator extends FlatSpec with Matchers with BeforeAndAfterAll
       "HllCardinality(client_count) as client_count",
       "HllCardinality(session_count) as session_count"
     ).collect()(0)
-    results.getAs[Any]("long_main_input_latency_session_count") should be (5)
-    results.getAs[Any]("client_count") should be (1)
-    results.getAs[Any]("session_count") should be (10)
+    results.getAs[Any]("long_main_input_latency_session_count") should be(5)
+    results.getAs[Any]("client_count") should be(1)
+    results.getAs[Any]("session_count") should be(10)
```

Comment:
Curious why these are getting caught in the diff..

Reply:
sneaky whitespaces